README.md

```diff
@@ -157,7 +158,8 @@ For more advanced code usage examples, please see example boards shown in [`exam
 | customSquareStyles            | object: {}                                                        | inline CSS styling                                 | Custom styles for all squares.                                                                                                                                                                                                                                                                                                                                                       |
 | id                            | number: 0                                                         | [string, number]                                   | Board identifier, necessary if more than one board is mounted for drag and drop.                                                                                                                                                                                                                                                                                                     |
 | isDraggablePiece              | function: ({ piece, sourceSquare }) => true                       | returns [true, false]                              | Function called when a piece drag is attempted. Returns if piece is draggable.                                                                                                                                                                                                                                                                                                       |
-| getPositionObject             | function: (currentPosition) => {}                                 |                                                    | User function that receives current position object when position changes.                                                                                                                                                                                                                                                                                                           |
+| getPositionObject             | function: (currentPosition) => {}                                 |                                                    | User function that receives current position object when position changes.
+| onArrowsChange              | function: (squares) => {}                                         |                                                    | User function is run when arrows are set on the board.  
```

Comment:
Needs formatting to match the rest of the table, as well as row ends please

Reply:
Done.